experiments/standalone/Bucket/global_bucket_staticmap.jl

```diff
@@ -85,43 +93,87 @@ z_0b = FT(1e-3);
 ρc_soil = FT(2e6);
 τc = FT(3600);
 t0 = 0.0;
-tf = 7 * 86400;
-Δt = 3600.0;
+tf = 14 * 86400;
+Δt = 3600.0 / 3;
 
 # Construct albedo parameter object using static map
 # Use separate regridding directory for CPU and GPU runs to avoid race condition
 device_suffix =
     typeof(ClimaComms.context().device) <: ClimaComms.CPUSingleThreaded ?
     "cpu" : "gpu"
-regrid_dir = joinpath(
-    pkgdir(ClimaLand),
-    "experiments/standalone/Bucket/$device_suffix/regrid-static/",
-)
-!ispath(regrid_dir) && mkpath(regrid_dir)
+t_start = t0
 surface_space = bucket_domain.space.surface
 α_snow = FT(0.8)
 albedo = PrescribedBaregroundAlbedo{FT}(α_snow, surface_space);
 
 bucket_parameters = BucketModelParameters(FT; albedo, z_0m, z_0b, τc);
 
+# Forcing data
+era5_artifact_path = "/groups/esm/ClimaArtifacts/artifacts/era5_land_forcing_data2021"# @clima_artifact("era5_land_forcing_data2021")
```

Comment:
```suggestion
era5_artifact_path = @clima_artifact("era5_land_forcing_data2021", context)
```

and on top you can `import ClimaUtilities: @clima_artifact`